web/src/components/vis/LoadingsPlotTile.vue

```diff
@@ -19,8 +22,8 @@ export default {
       type: Number,
       validator: Number.isInteger,
     },
-    points: {
-      validator: d => d === null || Array.isArray(d),
+    id: {
```

Comment:
why not adding the `id` property to the mixin?

Reply:
Because `id` doesn't need to be a prop for this mixin to work.  `id` could be a computed value or even just a data property.